### PR TITLE
feat!: Rename Form Field Character Counter to Character Count

### PR DESCRIPTION
**Pull request by RubenSibon**
**Breaking change!**

Rename the Form Field Character Counter to Form Field Character Count to be in line with [the NLDS reasoning](https://github.com/nl-design-system/backlog/issues/130#issuecomment-2114433450).

### CODE REVIEW

**Comment by RubenSibon** [comment]
8539517000_9a44748db4_b

File: packages/css/CHANGELOG.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 ### Features
 
 * Add Avatar component ([#1134](https://github.com/Amsterdam/design-system/issues/1134)) ([8dec2cf](https://github.com/Amsterdam/design-system/commit/8dec2cff8f7885ccb163774c0dc979fb2c781256))
-* Add Form Field Character Counter ([#1114](https://github.com/Amsterdam/design-system/issues/1114)) ([779ae8c](https://github.com/Amsterdam/design-system/commit/779ae8c9764b23508ffbfb38c6dc2f7a7c1cbd28))
+* Add Character Counter ([#1114](https://github.com/Amsterdam/design-system/issues/1114)) ([779ae8c](https://github.com/Amsterdam/design-system/commit/779ae8c9764b23508ffbfb38c6dc2f7a7c1cbd28))
```
Comment (VincentSmedinga) [inline packages/css/CHANGELOG.md:99]: I wouldn’t retroactively update the change log – this was the component's name when we introduced it. The next release will contain the title of this PR to indicate that it has changed.

File: packages/react/CHANGELOG.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 ### Features
 
 * Add Avatar component ([#1134](https://github.com/Amsterdam/design-system/issues/1134)) ([8dec2cf](https://github.com/Amsterdam/design-system/commit/8dec2cff8f7885ccb163774c0dc979fb2c781256))
-* Add Form Field Character Counter ([#1114](https://github.com/Amsterdam/design-system/issues/1114)) ([779ae8c](https://github.com/Amsterdam/design-system/commit/779ae8c9764b23508ffbfb38c6dc2f7a7c1cbd28))
+* Add Character Counter ([#1114](https://github.com/Amsterdam/design-system/issues/1114)) ([779ae8c](https://github.com/Amsterdam/design-system/commit/779ae8c9764b23508ffbfb38c6dc2f7a7c1cbd28))
```
Comment (VincentSmedinga) [inline packages/react/CHANGELOG.md:91]: Same.

File: proprietary/tokens/CHANGELOG.md [hunk 3]
```diff
@@ -81,7 +81,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 ### Features
 
 * Add Avatar component ([#1134](https://github.com/Amsterdam/design-system/issues/1134)) ([8dec2cf](https://github.com/Amsterdam/design-system/commit/8dec2cff8f7885ccb163774c0dc979fb2c781256))
-* Add Form Field Character Counter ([#1114](https://github.com/Amsterdam/design-system/issues/1114)) ([779ae8c](https://github.com/Amsterdam/design-system/commit/779ae8c9764b23508ffbfb38c6dc2f7a7c1cbd28))
+* Add Character Counter ([#1114](https://github.com/Amsterdam/design-system/issues/1114)) ([779ae8c](https://github.com/Amsterdam/design-system/commit/779ae8c9764b23508ffbfb38c6dc2f7a7c1cbd28))
```
Comment (VincentSmedinga) [inline proprietary/tokens/CHANGELOG.md:84]: Same

**Review by VincentSmedinga — CHANGES_REQUESTED** [review]

File: packages/css/CHANGELOG.md (hunk 1 above)
Comment (RubenSibon) [reply packages/css/CHANGELOG.md:99]: Aha, didn't intend to do that, CTRL-F'd to quickly.

**Review by VincentSmedinga — APPROVED** [review]

**Pull request merged**